libs/stats/tests/test_utils.py

```diff
@@ -26,6 +27,17 @@ def gen_compressed_dss(n,
         dt = dt + step
 
 
+def test_stac():
+    product = gm_product(location='/tmp/')
+    reader = TaskReader(str(TEST_DIR / 'test_tiles.db'), product)
+    task = reader.load_task(reader.all_tiles[0])
+
+    odc_meta = stac_transform(task.render_metadata())
```

Comment:
need to verify at least something about the content of `odc_meta`, for example lineage is kinda important, but also geometry and timestamps and location of files.

Reply:
Example output: https://gist.github.com/alexgleith/154959394a1ff44061b5d34a55301b54

Reply:
@alexgleith right and bottom a trimmed by fraction of a pixel

https://gist.github.com/alexgleith/154959394a1ff44061b5d34a55301b54#file-odc-from-stac-json-L304

I believe geometry should not be generated when it covers most of the image anyway, it's meant for defining valid pixel region and should not be present for 99% "full" images.